packages/web-react/src/components/Grid/__tests__/Grid.test.tsx

```diff
@@ -106,4 +98,52 @@ describe('Grid', () => {
     expect(element).not.toHaveStyle({ '--grid-spacing-x-desktop': 'var(--spirit-space-1200)' });
     expect(element).not.toHaveStyle({ '--grid-spacing-y-desktop': 'var(--spirit-space-1200)' });
   });
+
+  it('should render with default alignment classes', () => {
+    render(<Grid data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXStretch Grid--alignmentYStretch');
+  });
+
+  it('should render with horizontal alignment', () => {
+    render(<Grid alignmentX="center" data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXCenter');
+  });
+
+  it('should render with horizontal and vertical alignment', () => {
+    render(<Grid alignmentX="center" alignmentY="center" data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXCenter Grid--alignmentYCenter');
+  });
+
+  it('should render with some responsive horizontal alignments', () => {
+    render(<Grid alignmentX={{ tablet: 'center', desktop: 'right' }} data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass(
+      'Grid Grid--tablet--alignmentXCenter Grid--desktop--alignmentXRight',
+    );
+  });
```

Comment:
Cool, no we can see it 🙂.

We have a similar behavior in the `Stack` component, with one notable difference: when users define custom spacing on `tablet` or `desktop`, a default spacing is used for `mobile`. That's because there is a fallback value for custom property:

<img width="338" alt="obrazek" src="https://github.com/user-attachments/assets/7a17ab77-313b-47e3-a400-956441be9dca">

But in there are only CSS classes for the alignment of `Grid`.

We need the same behavior here so our API is consistent across components.

Cc @crishpeen: I can see two ways:

1. **JS logic:** if `alignment*` is an object and `mobile` key is not set, provide the default value (`stretch`).
2. **CSS magic:** something like
   ```
   .Grid:not(.Grid--alignment),
   .Grid--alignment<X/Y>Stretch {
     // …
   }
   ``` 
   🎉 

What do you guys think is better?

I think 1) better corresponds to what we said about the non-object defaults earlier.

Reply:
hmm, i added code to fix this issue:

react: [`41d08e7` (#1575)](https://github.com/lmc-eu/spirit-design-system/pull/1575/commits/41d08e70784fb12f5c6924d3319652bd43ed5f8e)
twig: [`442a5ff` (#1575)](https://github.com/lmc-eu/spirit-design-system/pull/1575/commits/442a5ff9dfdbc3bcb6c012c0bb5a31af54b3a507)

Reply:
I would do 1.

Reply:
me too, it is added to react and twig already